data/reserved_words_mysql.txt

```diff
@@ -211,6 +212,7 @@ full
 fulltext
 function
 general
+generate
```

Comment:
Not present in [MySQL 8.4 New Keywords and Reserved Words](https://dev.mysql.com/doc/refman/8.4/en/keywords.html#keywords-new-in-current-series),
but added in 8.0.32